de_identify_csv.py

```diff
@@ -39,6 +40,7 @@
 
     # Read the settings from the configuration file
     log.info("Loading Pipeline Configuration File...")
+    print(pipeline_configuration_file_path)
```

Comment:
Better to use log rather than print, consider adding it to the previous line

Reply:
This was a debug code that I forgot here. Deleted thank you.